packages/framework/src/Model/Customer/UserDataFactory.php

```diff
@@ -19,15 +19,58 @@ public function __construct(PricingGroupSettingFacade $pricingGroupSettingFacade
         $this->pricingGroupSettingFacade = $pricingGroupSettingFacade;
     }
 
+    /**
+     * @return \Shopsys\FrameworkBundle\Model\Customer\UserData
+     */
+    public function create(): UserData
+    {
+        return new UserData();
+    }
+
     /**
      * @param int $domainId
      * @return \Shopsys\FrameworkBundle\Model\Customer\UserData
      */
-    public function createDefault($domainId)
+    public function createFromDomainId(int $domainId): UserData
```

Comment:
I wanted to keep the naming `createFrom` and also `fillFrom`.

You may say that the data are filled from some internal dependency, so it doesn't make sense. But also when data object is created from an entity, it is usually filled by an internal dependency.

So... Do You still think this method should be named `createForDomainId`? If so, I'll change it

Reply:
for readability sake, I would prefer renaming the method to `createForDomainId` as the former seems quite confusing to me even though it will break the `createFrom*` naming consistency. I believe it is justified here :slightly_smiling_face: